app/src/ui/check-runs/ci-check-run-list-item.tsx

```diff
@@ -58,6 +74,25 @@ export class CICheckRunListItem extends React.PureComponent<ICICheckRunListItemP
     this.props.onViewJobStep?.(this.props.checkRun, step)
   }
 
+  private rerunJob = (e: React.MouseEvent) => {
+    e.stopPropagation()
+    if (this.props.checkRun.actionJobSteps === undefined) {
+      return
+    }
+
+    this.props.onRerunJob?.(this.props.checkRun)
+  }
+
+  private onMouseOver = () => {
```

Comment:
I will give it a shot, but I was thinking mouse enter only triggers on the div and mouse over also triggers on child elements.. but maybe it won't matter.

Reply:
Seems to work fine. For some reason I was thinking that it might not trigger all the time because of child elements taking up space, but this works just as well (and less events :)).

Reply:
Oh you're probably right. I was thinking mouseenter would bubble but it doesn't. Chances are `mouseover` will work just fine here but it's probably better to use your initial approach. Sorry for the distraction